### ISSUE TITLE
Bundle size too big...

### ISSUE THREAD

**Issue by olegskl** [wolfgangGoedel/RePeople#3]
I tried running the app with `webpack-dev-server --optimize-minimize --compress` and got bundle size of **1.2 MB** minified and gzipped...

**Comment by douglasduteil** [comment]
Seems normal as the webpack config and the imports are not optimized...

![vizu](https://user-images.githubusercontent.com/730511/31014507-7ea98cc4-a51b-11e7-9169-8ab75c7491cd.png)
(with [stats.json](https://gist.github.com/douglasduteil/8d0af7a0ecd5ab81fea9d82a653cedb1))
 
@wolfgangGoedel  you might want to use the [reasonml-community/reason-scripts](https://github.com/reasonml-community/reason-scripts)

**Comment by wolfgangGoedel** [comment]
yes - reason-scripts should add a bunch of useful configuration ;)
they didn’t exist when I started, but I’ll integrate them soon

**Comment by olegskl** [comment]
Indeed, it's not necessary to put all fonts in the bundle.

**Comment by RudyWeber** [comment]
The issue can be closed.
The project now uses `react-scripts`, thus giving a way lighter optimized production build :
```
File sizes after gzip:

  142.22 KB  build/static/js/main.c2b1f2f0.js
  22.69 KB   build/static/css/main.a7dcfc78.css
```
